app/services/check_course_jobs.rb

```diff
@@ -6,9 +6,27 @@
 class CheckCourseJobs
   include CourseQueueSorting
 
+  def self.remove_orphan_locks(courses_to_update)
```

Comment:
This is a good time to refactor this class as cleanly as you can. The `health_report` bit can be deleted, and you should figure out which method(s) are the public interface vs which are just going to be called internally, and use `private` to divide them.

Reply:
Done!